src/js/json5lint.js

```diff
@@ -24,40 +24,32 @@ fluid.defaults("fluid.lintAll.json5lint", {
  * Run the `json5lint` checks, i.e. ensure that all JSON5 files are valid.
  *
  * @param {Object} that - The `fluid.lintAll.json5lint` component.
- * @param {Array<String>} [checksToRun] - An array of check "keys" indicating which checks should be run.  If omitted,
- * all checks are run.
+ * @param {Array<String>} filesToScan - An array of files to check.
```

Comment:
I can't remember if our preferred form for these was String[] ?

Reply:
I think we've used a mix, for example here's one with the square notation in [core Infusion](https://github.com/fluid-project/infusion/blob/main/src/framework/core/js/Fluid.js#L460).  However, we also used `Array` on its own in the same file, instead of using square brackets.  I don't have a strong opinion about which syntax to use, I think we should:

1. Discuss at the next infusion-related meeting or on the chat/mailing list.
2. Agree on a convention.
3. Make it possible to check our work using [additional configuration options for the jsdoc plugin for ESLint](https://github.com/gajus/eslint-plugin-jsdoc/blob/master/test/rules/assertions/checkTypes.js#L661).